mirrulations-client/tests/test_client.py

```diff
@@ -211,6 +211,33 @@ def test_client_returns_403_error_to_server(mock_requests):
         assert '403' in response.json()
 
 
+def test_mock_get_timeout(mock_requests):
+    server_validator = Validator('http://test.com/')
+
+    with mock_requests:
+        mock_requests.get('http://test.com/http://test.com/get_results', exc=requests.exceptions.Timeout)
+        try:
+            server_validator.get_request('http://test.com/get_results')
+        except requests.exceptions.Timeout as error:
+            print(error)
```

Comment:
print statements aren't appropriate for tests.  Put a `pass` statement here and include a comment that this is the expected behavior.